new/LIT7091QeneAyte.xml

```diff
@@ -0,0 +1,89 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7091QeneAyte" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">አይ<corr>ቴ</corr>፡ ናስምክ፡ ውሉድ፡ አመጻ፡ ዘተመዝበረ፡ ልብነ፡ በግብረ፡ አመጻ፡ ነፋስ፤</title>
```

Comment:
again, I do not think editorial mark up or punctuation belong to the title of the record

Reply:
That is true, it is a title.